paasta_tools/cli/cmds/secret.py

```diff
@@ -90,12 +92,13 @@ def print_paasta_helper(secret_path, secret_name):
           "main:\n"
           "  cpus: 1\n"
           "  env:\n"
-          "    MY_SECRET_VAR: SECRET({})\n\n"
+          "    PAASTA_SECRET_MY_SECRET_VAR: SECRET({})\n\n"
```

Comment:
Thanks for doing this :-)

Do you think we should go a little further and nudge people to use the same name?

In other words change it to something like:
`PAASTA_SECRET_{}: SECRET({})\n\n".format(secret_name.upper(), secret_name)`